src/app/loadout-builder/generated-sets/GeneratedSet.m.scss

```diff
@@ -25,6 +26,19 @@
   }
 }
 
+.existing-loadout {
+  margin: 0;
+  color: #e8a534;
+
+  @media (min-width: 540px) {
```

Comment:
I'm not sure if we need this query anymore? Margin is set to 0 in the main class.